vendor/licenses/grammar/language-pcb.txt

```diff
@@ -1,10 +1,25 @@
 ---
 type: grammar
 name: language-pcb
-version: 3d27661a97408f30a68591c5f43b184652aac76f
+version: 676367a91d7664caa510b44df7d2cd0c7685bb86
 license: isc
 ---
-Copyright (c) 2017, John Gardner
+Copyright (c) 2017-2018, John Gardner
```

Comment:
Why was the license text doubled here?

Reply:
Oh, I didn't notice that. I ran `script/licensed` to force-update the license hash after CI complained. 

Which, erm, I'm guessing wasn't the correct procedure. Then again, submodules aren't supposed to be "bumped" like this, so... 😕